helm/applications/skaha/init-users-datasets-symlink-config/init-users-datasets-symlink.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [ -d "/datasets" ]; then
```

Comment:
Doesn't this need to be configured to what the `deployment.skaha.sessions.prepareData.datasetRootPath` is set to?